networking/networking.tex

```diff
@@ -768,20 +768,20 @@ \section{Layer 4: TCP Server}
   \end{lstlisting}
 
   One can use the macros \keyword{NI\_MAXHOST} to denote the maximum length of a hostname, and \keyword{NI\_MAXSERV} to denote the maximum length of a port.
-  \keyword{NI\_NUMERICHOST} gets the hostname as a numeric IP address and similarly for \keyword{NI\_NUMERICSERV} though the port is usually numeric to begin with.
+  \keyword{NI\_NUMERICHOST} gets the hostname as a numeric IP address and similarly for \keyword{NI\_NUMERICSERV} although the port is usually numeric, to begin with.
   The \href{https://man.openbsd.org/getnameinfo.3#NI\_NUMERICHOST}{Open BSD man pages have more information}
 
   \item \keyword{int close(int fd)} and \keyword{int shutdown(int fd, int how)}
 
   Use the \keyword{shutdown} call when you no longer need to read any more data from the socket, write more data, or have finished doing both.
-  When you shutdown a socket for further writing or reading, that information is also sent to the other end of the connection.
-  If you shutdown the socket for further writing at the server end, then a moment later, a blocked \keyword{read} call could return 0 to indicate that no more bytes are expected.
-  Similarly, a write to a TCP connection that has been shutdown for reading will generate a SIGPIPE
+  When you shut down a socket for further writing or reading, that information is also sent to the other end of the connection.
```

Comment:
This should've been wrapped in the system call \keyword{shutdown}.
The proper phase is `When you call \keyword{shutdown}` on a socket.